SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTests.cs

```diff
@@ -11,6 +11,7 @@ namespace SIL.Windows.Forms.Tests.Miscellaneous
 	class PortableClipboardTests
 	{
 		private const string TestImageDir = "SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTestImages";
+		private static readonly object _lock = new object();
```

Comment:
awesome, I'm glad to see this worked. Can you put a comment explaining why we need this lock, and reference the error we were seeing?

Reply:
Done